app/src/main/java/org/jellyfin/androidtv/constant/CodecTypes.java

```diff
@@ -5,16 +5,20 @@
     public static final String AAC = "aac";
     public static final String AAC_LATM = "aac_latm";
     public static final String AC3 = "ac3";
+    public static final String ALAC = "alac";
     public static final String APE = "ape";
     public static final String DCA = "dca";
     public static final String DTS = "dts";
     public static final String EAC3 = "eac3";
     public static final String FLAC = "flac";
+    public static final String MLP = "mlp";
     public static final String MP2 = "mp2";
     public static final String MP3 = "mp3";
     public static final String MPA = "mpa";
     public static final String OPUS = "opus";
     public static final String PCM = "pcm";
+    public static final String PCM_ALAW = "pcm_alaw";
+    public static final String PCM_MULAW = "pcm_mulaw";
     public static final String PCM_S16LE = "PCM_S16LE";
```

Comment:
I'm not sure if it matters, but the android app has codec strings in lowercase. This might be important to the server but idk.

https://github.com/jellyfin/jellyfin-android/blob/ade07dbb62c652bb661fa29b931d3372fed01047/app/src/main/java/org/jellyfin/mobile/api/DeviceProfileBuilder.kt#L263